src/CentreonRemote/Domain/Exporter/ConfigurationExporter.php

```diff
@@ -80,8 +80,10 @@ public function import(ExportManifest $manifest): void
 
             $import = $manifest->get("import");
             foreach ($import['data'] as $data) {
-                // truncate table
-                $db->query("TRUNCATE TABLE `" . $data['table'] . "`;");
+                // empty table
+                $db->query("DELETE FROM `" . $data['table'] . "`");
+                // optimize table
+                $db->query("ALTER TABLE `" . $data['table'] . "`");
```

Comment:
Can you expected the expected side effect of this query ? Why not use a clearer query like `OPTIMIZE TABLE` or `ANALYSE TABLE` ?

Reply:
I've updated to OPTIMIZE TABLE
I thought ALTER TABLE rebuild data and index too. But regarding documentation, it does not..